app/frontend/src/locales/es/translation.json

```diff
@@ -81,6 +81,10 @@
         "minimumSearchScore": "Puntaje mínimo de búsqueda",
         "minimumRerankerScore": "Puntaje mínimo de re-clasificación",
         "retrieveCount": "Obtén éste número resultados de búsqueda:",
+        "includeCategory": "Incluir categoría",
```

Comment:
Let me know if any of these translations require a human review.

Reply:
They do need a human review, please.

Reply:
Changes made based on the review.